packages/notifier/README.md

```diff
@@ -94,55 +102,54 @@ observeIteration(subscription, observer);
 // finished done
 ```
 
-The difference between the two pairs is that the NotifierKit is a lossy
-conveyor of non-final values whereas the SubscriptionKit is a lossless conveyor
-of non-final values. They both losslessly convey termination, so the
-SubscriptionKit losslessly conveys the entire async iteration.
+Note that SubscriptionKit is a *lossless conveyor* of non-final values. It conveys all of 
+an async iteration’s non-final values, as well as the termination (if not necessarily the 
+actual final value).
 
-Had the example above started with
+On the other hand, NotifierKit is a *lossy conveyor* of non-final values, but does also 
+losslessly convey termination. Had the example above started with the following instead 
+of using `makeSubscriberKit()`, 
 ```js
 const { updater, notifier } = makeNotifierKit();
 ```
-propagating the variable renaming through the rest of the code, the code above
-remains correct, but Alice and Bob may each have missed either or both of the
-non-final values.
-
+...and had the different variables (`updater` and `notifier` replacing `publication` 
+and `subscription`) renamed through the rest of the code, the code is still correct. However, 
```

Comment:
Too many words for the renaming point, since it is just stating something obvious for correctness. Maybe

```suggestion
(renaming `publication` to `updater` and `subscription` to `notifier` in the rest of the code), the code is still correct. However, 
```